### DISCUSSION TITLE
Option to add/load specific type-stubs from PyPI

### DISCUSSION THREAD

**Discussion by Josverl** [erictraut/pyright-playground#1] [Q&A] [answered]
Hi,
Very nice to be able to see this in the browser so people can see the effects before changing their dev setup.

Would it be possible to load / install a typestub package from PyPI that is not included by default in Pyright/Pylance ?
or would that require a fork and changes to the build process ?

**Comment by erictraut** [comment, marked as answer]
That's not possible currently. It's something we could potentially add in the future, but I think it's unlikely that we'd do so. It goes beyond what the playground is intended for. Installing arbitrary packages from PyPi in the playground environment would have both performance and security implications. The other playground environments that this one is modeled after likewise do not allow installation of other packages.

**Reply by Josverl** [reply]
OK, a pity but I can understand your reasoning.

Thanks for the clarity.
